### DISCUSSION TITLE
How to change page width?

### DISCUSSION THREAD

**Discussion by purbaneck** [luizdepra/hugo-coder#835] [Q&A] [answered]
Hi!

First of all: Thanks for providing this cool Hugo theme! 

I'm trying to change the standard page width but I cannot figure out how (sorry, I'm not very experienced in HTML/CSS). The issue is that I have too many menu items and the page title on the left slides up a bit so page title and menu items are not in one line anymore. That doesn't hurt in the mobile view as menu items are collapsed anyhow but it doesn't look good in desktop view. 

I've changed each max-width lines in all CSS-files to 1024px but that doesn't change anything. Can't figure out how to solve that. Sorry, as I'm positive that this might be a beginners question.

Best,
Pat

**Comment by luizdepra** [comment, marked as answer]
Hey.

You need to change the `max-width` value from `.container`. The default value is `90 rem`. Try increasing it.
REM is a relative value, based on the root font size.

**Reply by purbaneck** [reply]
Perfect, that worked just fine! Thank you so much!
